projects/demo/src/modules/app/pages.ts

```diff
@@ -284,6 +284,12 @@ export const pages: TuiDocPages = [
                 keywords: `dropdown, контекст, выпадашка, дропдаун, Context`,
                 route: `/directives/dropdown`,
             },
+            {
+                section: `Tools`,
+                title: `DropdownOpen`,
+                keywords: `dropdown, контекст, выпадашка, дропдаун, меню, menu`,
```

Comment:
Should we add keyword "hosted" (for users who are already used to the previous name)?